app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,28 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @current = current_user
+  end
+
+  def create
```

Comment:
Please see the error below which occurs when the user tries to create a new post. The post is created but the user is not redirected to any other path.
![new_post](https://user-images.githubusercontent.com/35504149/155699432-797372c2-f2cc-4f62-bf3c-95c38e4a60f0.png)

Reply:
resolved

Reply:
- [ ] While you tried to resolve the above error, I new one was created and a post cannot still be created. I got the error below. Kindly fix it and ensure that you test from your end that a post can be created successfully.
![Screenshot from 2022-02-25 20-41-12](https://user-images.githubusercontent.com/20245164/155763306-1a0741cd-99fc-49a1-806f-d1a3fe042eb6.png)